packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.js

```diff
@@ -174,6 +174,12 @@ export default class QuanticDateFacet extends LightningElement {
     return '';
   }
 
+  fixDateForAllBrowsers(dateString) {
+    return dateString
+      .replaceAll('/', '-')
+      .replaceAll('@', 'T');
+  }
```

Comment:
Why do we use an obscure format for dates instead of using the ISO-8601 standard for string representation (not locale representation) ? @olamothe @samisayegh

Reply:
Hi Jérémie, we use the format used by the [search api](https://docs.coveo.com/en/1461/build-a-search-ui/query-parameters#RestFacetValue-start).

Headless could have transformed the dates to ISO, but the thinking was to act as an api client and simply pass-through the data. One detail is the api only supports a single timezone, that can be set when instantiating an engine. All facet values use that timezone, i.e. it's not possible to independently vary the zone as ISO might allow.